tests/test_yamls/test_storage_mounting.yaml.j2

```diff
@@ -27,6 +27,14 @@ file_mounts:
     source: ~/tmp-workdir
     mode: MOUNT
 
+  # Mounting a private bucket in CSYNC mode
+  /mount_private_csync:
+    name: {{csync_storage_name}}
+    source: ~/tmp-workdir-2
+    mode: CSYNC
+    interval: 5
```

Comment:
Might be better to include the unit in the name. How about `interval_minutes`?

Reply:
Those are in seconds, so I'll update for `interval_seconds`.